zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/deployment/DbProcessState.java

```diff
@@ -94,6 +94,19 @@ public final class DbProcessState implements MutableProcessState {
           DbForeignKey<DbTenantAwareKey<DbLong>>>
       processDefinitionKeyByProcessIdAndDeploymentKeyColumnFamily;
 
+  private final DbString versionTag;
+  private final DbTenantAwareKey<DbCompositeKey<DbString, DbString>>
+      tenantAwareProcessIdAndVersionTagKey;
+
+  /**
+   * <b>Note</b>: Will only be filled with processes deployed from 8.6 onwards that have a version
+   * tag assigned (which is an optional property).
+   */
```

Comment:
❓ Does this mean that we don't need to perform a migration on already deployed processes?

🤔 I wonder if it would be good to have a migration test to confirm that there are no migration issues due to this change.

Reply:
I'm not really familiar with the migrations, but in my opinion, we don't need to perform one, because already deployed processes won't have a version tag anyway.

By the way, I also had a similar discussion with Nico in the context of "deployment version binding" and we concluded that a migration would not be necessary (see https://github.com/camunda/camunda/issues/19709#issuecomment-2245723417 and the comments above).

Reply:
Ah okay, that's good to see. Then consider this discussion resolved 😄